### PR TITLE
adding GEM/webnlg prompts 

### PR DESCRIPTION
**Pull request by jordiclive**
GEM/webnlg houses all constituents, and challenge sets. Also added Russian prompts with minimal changes, as input triple set is still english.

### CODE REVIEW

**Comment by jordiclive** [comment]
I've added Multi-refs

**Comment by jon-tow** [comment]
Hey @jordiclive ! When I run this I get a `List[str]` as opposed to the expected `Tuple[str, List[str]` return (see https://github.com/bigscience-workshop/promptsource/pull/747).

Example: For `GEM/web_nlg` `ru`

```python
import datasets
from promptsource.templates import DatasetTemplates

template = DatasetTemplates("GEM/web_nlg", "ru")
data = datasets.load_dataset("GEM/web_nlg", "ru")
example = data['validation'][0]

prompt = template['PALM_prompt'].apply(example)
print(prompt)
```

```
[
'I will verbalize an abstract representation of a sentence in natural language. To do so, I will first show the representation and then the natural language. The text needs to include all of the information in the representation and it will be in Russian.\n\nPunjab,_Pakistan | leaderTitle | Provincial_Assembly_of_the_Punjab',
'Пенджаб, Пакистан, возглавляется Провинциальной ассамблеей Пенджаба.',
'Пенджаб, Пакистан возглавляется Провинциальной ассамблеей Пенджаба.'
]
```

**Comment by jordiclive** [comment]
@jon-tow, I think you were not working with the newest eval-hackathon which has the multi-ref changes. I have now merged that into this .

**Comment by sebastianGehrmann** [comment]
Prompts look good to me. Do we have someone who could translate some of the Russian prompts so we don't only rely on English?

File: promptsource/templates/GEM/web_nlg/en/templates.yaml [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+      {{input | join(\", \")}} {% for i in references %}\n  ||| {{ i }} \n{% endfor\
+      \ %}"
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - BLEU
+      - ROUGE
+      original_task: true
+    name: PALM_prompt
+    reference: used-in-PALM
```
Comment (Muennighoff) [inline promptsource/templates/GEM/web_nlg/en/templates.yaml:33]: how do we know this prompt was used in palm?
all i could find in their paper is the below ..
Screenshot 2022-08-15 at 22 11 45

Comment (jordiclive) [reply promptsource/templates/GEM/web_nlg/en/templates.yaml:33]: I believe @sebastianGehrmann shared some NLG prompts for PALM.

**Pull request merged**